### ISSUE TITLE
About remesher of MeshGraphNets ?

### ISSUE THREAD

**Issue by chufall** [deepmind/deepmind-research#519]
Hi, I'm reading the paper "LEARNING MESH-BASED SIMULATION WITH GRAPH NETWORKS".
In the section 3.2 "ADAPTIVE REMESHING", the paper metioned "At test time, for each time step we predict both the next simulation state and the sizing ﬁeld, and use a generic, domain- independent remesher R to compute the adapted next-step mesh".
There're two questions:
1. Does the remeshing operation will changed the number of nodes? since one of the remeshing is spliting which will create a new node?
2. Will it perform the remeshing operation when traing the model? 

Thanks a lot!
qc

**Comment by ambreshbiradar9** [comment]
Were you able to find data set ?
![data](https://github.com/google-deepmind/deepmind-research/assets/19732486/71350981-ec1d-49e1-b2d9-947d4394a587)

**Comment by ambreshbiradar9** [comment]
Hi 
I want to understand this paper.
Please can you help me 
My email id: ambreshbiradar9@gmail.com

**Comment by bazziamir24** [comment]
1- The remeshing operation will change the number of nodes.
2- Yes it is fed into the loss function.

**Comment by paulaguti** [comment]
> 1- The remeshing operation will change the number of nodes. 2- Yes it is fed into the loss function.

I am wondering how do they compute the loss function against the target as, indeeed, the number of nodes nodes change for the prediction during training, but not for the target. Do they interpolate the target values or how do the compute compute the new "ground truth fields" for the new nodes?
